core/src/main/java/io/javaoperatorsdk/jenvtest/KubeAPIServerConfig.java

```diff
@@ -20,10 +21,19 @@ public class KubeAPIServerConfig {
    */
   private final boolean offlineMode;
 
-  KubeAPIServerConfig(String jenvtestDir, String apiServerVersion, boolean offlineMode) {
+  /**
+   * Flags to pass to Kube API Server on startup. Key and value are two separated items, like
+   * specifying min-request-timeout needs to add in order two values: "--min-request-timeout" and
+   * "300" for the actual desired value.
+   */
+  private final List<String> apiServerFlags;
+
+  KubeAPIServerConfig(String jenvtestDir, String apiServerVersion, boolean offlineMode,
+      List<String> apiServerFlags) {
```

Comment:
Maybe it could be useful to have a `Flag` class that would encapsulate things instead of just relying on String pairs (similarly to what's done for micrometer tags)?

Reply:
Ahh not sure, it is also some time not a pair, just a single flag name